examples/metrics.json

```diff
@@ -81,11 +81,11 @@
                   {
                     "startTimeUnixNano": 1544712660300000000,
                     "timeUnixNano": 1544712660300000000,
-                    "count": 3,
-                    "sum": 3,
-                    "bucketCounts": [1,1,1],
+                    "count": 2,
+                    "sum": 2,
+                    "bucketCounts": [1,1],
                     "explicitBounds": [1],
-                    "min": 1,
+                    "min": 0,
                     "max": 1,
```

Comment:
If the count is 2, and the min is zero, then one of the two meaurements is zero. Therefore for the sum to be 2, the other measurement must be 2.

Reply:
Good catch, fixed.